src/scripts/create-parameters.py

```diff
@@ -38,10 +38,7 @@
   capture_output=True
 ).stdout.decode('utf-8').splitlines()
 
-mappings = [
-  m.split() for m in
-  os.environ.get('MAPPING').splitlines()
-]
+mappings = [m.split() for m in os.environ.get("MAPPING").splitlines() if m.strip()]
```

Comment:
it will be better to strip all mapped lines first, regardless.

```suggestion
mappings = [
    m.split() for m in [n.strip() for n in os.environ.get("MAPPING").splitlines()]
]
```

Reply:
In my changes, the lines are stripped first, regardless, and empty lines are filtered, while it's not the case in the suggested changes, as we'll end up splitting empty lines, thus, not fixing the error.

Reply:
You can try it locally

```python
>>> s = """a b false
... c d true
...
... e f false"""
>>> mappings = [m.split() for m in [n.strip() for n in s.splitlines()]]
[['a', 'b', 'false'], ['c', 'd', 'true'], [], ['e', 'f', 'false']] # empty list : /
>>> filter(check_mapping, mappings)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 3, in check_mapping
Exception: Invalid mapping
```

Reply:
@taoufik07 

ah, that is true, since `if m.strip()` will always be evaluated (run) in your code.
that looks good to me!
thank you for the correction 👍 👍